src/shogun/base/init.cpp

```diff
@@ -37,6 +38,9 @@ namespace shogun
 	Version* sg_version=NULL;
 	CMath* sg_math=NULL;
 	CRandom* sg_rand=NULL;
+#ifdef HAVE_CXX11
```

Comment:
do we need this? i mean wasn't the idea for post GSOC stuff was that we will require c++11?